### PR TITLE
Feat: Linked Wearables Category

### PR DESCRIPTION
**Pull request by ncomerci**
This PR addresses [this issue](https://www.notion.so/dcl-dao/New-Proposal-Category-for-Linked-Wearables-9733bfe6360e4955ade49d87a40a94cf)

**IMPORTANT**
- New environment variable added called: `GATSBY_VOTING_POWER_TO_PASS_LINKED_WEARABLES`
- Passage Threshold: 4M VP
- Voting Period: 1 Week
- "New" badge added for future new categories on submit page

## Views
screencapture-localhost-8000-submit-2022-02-16-10_40_06
screencapture-localhost-8000-submit-linked-wearables-2022-02-16-10_40_28
screencapture-localhost-8000-proposal-2022-02-16-11_17_24
screencapture-localhost-8000-2022-02-16-10_41_57

### CODE REVIEW

File: src/entities/Proposal/routes.ts [hunk 1]
```diff
@@ -339,6 +340,31 @@ export async function createProposalGrant(req: WithAuth) {
...
+export async function createProposalLinkedWearables(req: WithAuth) {
+  const user = req.auth!
+  const configuration = validate<NewProposalLinkedWearables>(newProposalLinkedWearablesValidator, req.body || {})
+
+  req.body.links = req.body.links.map((link: string) => {
+    if(link.startsWith('http')) {
+      return link
+    }
+    
+    return 'https://' + link
```
Comment (andyesp) [inline src/entities/Proposal/routes.ts:353]: I think you can add this validation in the input field
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

**Review by andyesp — APPROVED** [review]

File: src/pages/submit/linked-wearables.tsx [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+      ),
+      governance: (assert(state.governance.length > 0, 'error.linked_wearables.governance_empty') ||
+      assert(state.governance.length >= schema.governance.minLength, 'error.linked_wearables.governance_too_short') ||
+      assert(state.governance.length <= schema.governance.maxLength, 'error.linked_wearables.governance_too_large')
+      ),
+      motivation: (assert(state.motivation.length > 0, 'error.linked_wearables.motivation_empty') ||
+      assert(state.motivation.length >= schema.motivation.minLength, 'error.linked_wearables.motivation_too_short') ||
+      assert(state.motivation.length <= schema.motivation.maxLength, 'error.linked_wearables.motivation_too_large')
+      ),
+      method: (assert(state.method.length <= schema.method.maxLength, 'error.linked_wearables.method_too_large')
```
Comment (1emu) [inline src/pages/submit/linked-wearables.tsx:129]: I think we are missing the method too short validation

**Review by 1emu — APPROVED** [review]
Left a comment about a validation, the rest looks good! ⭐

File: src/pages/submit/linked-wearables.tsx (hunk 2 above)
Comment (andyesp) [reply src/pages/submit/linked-wearables.tsx:129]: It's missing because it's optional 🤔

**Review by andyesp — APPROVED** [review]

**Pull request merged**